release-notes/opensearch-dashboards.release-notes-2.0.0-rc1.md

```diff
@@ -0,0 +1,93 @@
+## Version 2.0.0-rc1 Release Notes
+
+### Breaking Changes in 2.0
+
+#### Bump to node version 14.18.2
+* [Node 14] Upgrades Node version to 14.18.2 ([#1028](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1028))
+
+#### Use opensearch-project/opensearch as nodejs client
+* [nodejs client] hookup js client with dashboards ([#1342](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1342))
+* [nodejs client] modify tsconfig to route types to new.d.ts ([#1225](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1255))
+
+#### Disable telemetry by default
+* Fixes search usage telemetry ([#1427](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1427))
+
+### UI library
+* Removes UI Framework KUI doc site ([#1379](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1379))
+* Bumps @elastic/eui to v34.6.0 and @elastic/charts to v31.1.0 ([#1370](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1370))
+
+#### Deprecations
+* Deprecates non-inclusive config names ([#1467](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1467))
+* Removes unnecessary manual resolutions ([#1300](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1300))
+* Removes backport and release-notes scripts ([#1234](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1234))
+* Removes storybook package and related code ([#1172](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1172))
+
+### Security Fixes
+* [CVE] Bumps Node.js from v14.18.2 to v14.19.1 ([#1487](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1487))
```

Comment:
I think we should list out all of the individual CVEs that we addressed with each commit. Can we have sub-bullets for each commit if there are multiple CVEs? I can help create that list if you need help!

Reply:
will do